pkg/list/list_deployments_test.go

```
@@ -0,0 +1,1501 @@
+// mockDescribeStacks is a mock implementation of describe.ExecuteDescribeStacks.
+type mockDescribeStacks struct {
+	stacks map[string]interface{}
+	err    error
+}
+
+func (m *mockDescribeStacks) Execute(config *schema.AtmosConfiguration, stack string, additionalStacks []string, additionalStacksList []string, additionalStacksMap map[string]bool, includeAllStacks bool, includeStackGroups bool) (map[string]interface{}, error) {
+	return m.stacks, m.err
+}
+
+// isDriftDetectionEnabled checks if drift detection is enabled for a deployment.
+func isDriftDetectionEnabled(deployment *schema.Deployment) bool {
```

golangci-lint (1.64.8)

[error] func `isDriftDetectionEnabled` is unused

(unused)